andela/balrog
# bug 1337929: make it possible to link to the Rules page with a specific filter

#### What does this PR do?
makes it possible to link to the Rules page with a specific filter
#### Description of Task to be completed?
make it possible to link to the Rules page with a specific filter
#### How should this be manually tested?
once balrog is running on http://127.0.0.1:8080, navigate to rules and select a single rule on the Product/Channel dropdown. The URL should change to match the selected product.
#### Any background context you want to provide?
When we used a free text filter, we were able to send links to the Rules page with a filter applied. Since we switched to the drop down, we haven't been able to do that anymore. This PR remedies that by  reflecting the value of the current drop down filter in the URL.

#### What are the relevant bugzilla stories?
https://bugzilla.mozilla.org/show_bug.cgi?id=1337929
#### Screenshots (if appropriate)
#### Questions:
